tests/Fixtures.h

```diff
@@ -0,0 +1,76 @@
+#pragma once
+#include "UsgsAstroFramePlugin.h"
+#include "UsgsAstroFrameSensorModel.h"
+
+#include <json.hpp>
+
+#include <sstream>
+#include <fstream>
+
+#include <gtest/gtest.h>
+
+using json = nlohmann::json;
+
+// Should this be positioned somehwere in the public API?
+inline void jsonToIsd(json &object, csm::Isd &isd, std::string prefix="") {
+  for (json::iterator it = object.begin(); it != object.end(); ++it) {
+     json jsonValue = it.value();
+
+     // Is this type of type sensative de-serialization not available
+     // out of the box?
+     if (jsonValue.is_object()) {
+       // Is flattening the best approach? not sure...
+       jsonToIsd(jsonValue, isd, prefix+it.key()+'_');
```

Comment:
I don't think we want to flatten the JSON. What we can do is convert the object into a JSON encoded string and then just pass that as the ISD parameter value. The plugin can then parse out the object.